src/reporting/Interfaces.sol

```diff
@@ -34,12 +33,22 @@ contract IMarket is Typed {
     function isDoneWithAllReporters() constant public returns (bool);
     function isDoneWithLimitedReporters() constant public returns (bool);
     function getReportingTokenOrZeroByPayoutDistributionHash(bytes32) constant public returns (ReportingToken);
+    function getFinalizationTime() constant public returns (uint);
+    function getDenominationToken() constant public returns (ERC20);
+    function getPayoutDenominator() constant public returns (int256);
```

Comment:
This should (will) return `uint256`.  In Serpent, `int256` is really the only type which is why everything is `int256`.  As we have been migrating to Solidity, we have been adjusting things to their correct types and in this case that is unsigned.